libraries/chain/include/eosio/chain/contract_types.hpp

```diff
@@ -156,6 +156,24 @@ struct providebw {
     }
 };
 
+struct requestbw {
```

Comment:
It's need to add description about this action into default blockchain ABI-description too. This description will be used when `eosio` contract doesn't load yet.